tests/test_peak_time_range.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed May 31 10:04:56 2023
+
+@author: claudia
+"""
+
+import os
+import pytest
+import numpy as np
+import random
+import math
+from scipy.stats import norm
+
+from ramp.core.core import User, Appliance
+from ramp.core.core import UseCase
+from ramp.core.initialise import initialise_inputs
+from ramp.core.stochastic_process import calc_peak_time_range
```

Comment:
```suggestion
import numpy as np

from ramp.core.initialise import initialise_inputs
from ramp.core.stochastic_process import calc_peak_time_range
```
In the suggestion I removed the unused imports